### PR TITLE
Add list view

### PR DESCRIPTION
**Pull request by sokolikp**

### CODE REVIEW

File: Sweeper/Storyboards/Pinviews.storyboard [hunk 1]
```diff
@@ -1,58 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sokolikp) [inline Sweeper/Storyboards/Pinviews.storyboard:1]: This is just a rename.

File: Podfile [hunk 2]
```diff
@@ -6,14 +6,15 @@ target 'Sweeper' do
...
-  pod 'Parse', '~> 1.15.3'
+  pod 'Parse'
   pod 'KMPlaceholderTextView', '~> 1.3.0'
   pod 'AFNetworking'
 
   # AWS SDKs
   pod 'AWSCognito'
   pod 'AWSCore'
   pod 'AWSS3'
+  pod 'NSDateMinimalTimeAgo'
```
Comment (wumingx89) [inline Podfile]: We should use SwiftDate

File: Sweeper/ViewControllers/LoginViewController.swift [hunk 3]
```diff
@@ -108,8 +108,8 @@ class LoginViewController: UIViewController {
     }
     
     private func segueToHome() {
-        let storyboard = UIStoryboard(name: "Pinviews", bundle: nil)
-        let navigationVC = storyboard.instantiateViewController(withIdentifier: "PinviewsNavigationController")
+        let storyboard = UIStoryboard(name: "PinViews", bundle: nil)
```
Comment (wumingx89) [inline Sweeper/ViewControllers/LoginViewController.swift]: Can you add this to the Storyboard extension?

**Review by huw30 — APPROVED** [review]
lgtm

File: Podfile (hunk 2 above)
Comment (wumingx89) [inline Podfile]: Let's use SwiftDate

**Review by wumingx89 — COMMENTED** [review]
I think we should also pull the pin cell to a nib

**Review by wumingx89 — APPROVED** [review]

**Pull request merged**